docs-2.0/5.configurations-and-logs/2.log-management/logs.md

```diff
@@ -21,7 +21,7 @@ The default severity level for the metad, graphd, and storaged logs can be found
 
 ## Check and Change the Severity Levels Dynamically
 
-Check all the flag values (log values included) of the current gflags with the following command. `curl` is only available when the `local_config` parameter is set to `true`.
+Check all the flag values (log values included) of the current gflags with the following command. `curl` is only available when the `local_config` parameter is set to `true`. Not all flags are listed because changing some flags can be dangerous. Read the response explanation and the source code before you change these not documented parameters. To get all the available flags for a process, use this command:
```

Comment:
`curl` is only available when the `local_config` parameter is set to `true`. -- delete this sentence.  It can work whether true or false.